src/components/Accordion.tsx

```diff
@@ -23,6 +23,8 @@ export interface AccordionProps {
   setExpandedIndex?: Dispatch<SetStateAction<number | undefined>>;
   /** Used by Accordion list. Sets default padding to 0 for nested accordions */
   omitPadding?: boolean;
+  /** Styles overrides for omit padding */
+  xss?: Xss<Padding>;
 }
 
 export function Accordion(props: AccordionProps) {
```

Comment:
**request** In order for the XSS to property to only allow `Padding`, then we need to add some more to this component's function definition.
```
type AccordionXss = Xss<Padding>

export interface AccordionProps<X> {
  xss?: X;
}

export function Accordion<X extends Only<AccordionXss, X>>(props: AccordionProps<X>) {
```